src/test/steps/loginSteps.ts

```diff
@@ -1,37 +1,37 @@
 import {Given, When, Then} from '@cucumber/cucumber';
-import {chromium, Page, Browser, expect} from '@playwright/test';
+import {expect} from '@playwright/test';
+import basePage from '../../pages/basePage';
 
-let browser: Browser;
-let page: Page;
+Given('User passes the authorization', async function () {
+  await basePage.authPage.authorizeUser();
 
-Given('User navigates to the application', async function () {
-  browser = await chromium.launch({headless: false});
-  page = await browser.newPage();
-  page.goto('https://px-stage.self-service.danads.com');
+  const pageTitle = await basePage.page.title();
+  expect(pageTitle).toContain('Core Ad Manager');
 });
 
-Given('User clicks login button', async function () {
-  //await page.locator("//a/*[text()='Log in']").click();
+Given('User clicks Log in button', async function () {
+  await basePage.wrapper.waitAndClick(basePage.loginPage.logInBtn);
 });
 
-Given('User enters username', async function () {
-  await page.locator('(//input[@name=\'username\'])[2]').fill('Username');
+When('User enters email and password as a {string}', async function (userRole: string) {
+  basePage.loginPage.enterUsernameAndPassowrd(userRole);
 });
 
-When('User enters log in with email button', async function () {
-  //page.locator("//span[text()=' Log in with email ']").click();
+/* When('User clicks Log in with email button', async function () {
+  await basePage.wrapper.waitAndClick(basePage.loginPage.submitBtn);
 });
 
 When('User enters password', async function () {
-  await page.locator('(//input[@name=\'password\'])[2]').fill('Password');
-});
+  await basePage.wrapper.type(basePage.loginPage.passwordInput, process.env.ADMIN_PASSWORD);
+}); */
```

Comment:
Don't leave comments unless it is necessary!

Reply:
Fixed